<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="1" failed="1" ignored="0" total="4" passed="2">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="37195" started-at="2017-10-17T09:37:12Z" finished-at="2017-10-17T09:37:49Z">
    <groups>
      <group name="methodgroup1">
        <method signature="Dependency.openGmail()[pri:0, instance:package1.Dependency@5e5792a0]" name="openGmail" class="package1.Dependency"/>
        <method signature="Dependency.login()[pri:1, instance:package1.Dependency@5e5792a0]" name="login" class="package1.Dependency"/>
        <method signature="Dependency.ending()[pri:3, instance:package1.Dependency@5e5792a0]" name="ending" class="package1.Dependency"/>
      </group> <!-- methodgroup1 -->
    </groups>
    <test name="Default test" duration-ms="37195" started-at="2017-10-17T09:37:12Z" finished-at="2017-10-17T09:37:49Z">
      <class name="package1.Dependency">
        <test-method status="PASS" signature="openGmail()[pri:0, instance:package1.Dependency@5e5792a0]" name="openGmail" duration-ms="6952" started-at="2017-10-17T15:07:12Z" finished-at="2017-10-17T15:07:19Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- openGmail -->
        <test-method status="FAIL" signature="login()[pri:1, instance:package1.Dependency@5e5792a0]" name="login" duration-ms="30139" started-at="2017-10-17T15:07:19Z" depends-on-methods="package1.Dependency.openGmail" finished-at="2017-10-17T15:07:49Z">
          <exception class="org.openqa.selenium.NoSuchElementException">
            <message>
              <![CDATA[Unable to locate element: {"method":"id","selector":"Email"}
Command duration or timeout: 30.05 seconds]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.openqa.selenium.NoSuchElementException: Unable to locate element: {"method":"id","selector":"Email"}
Command duration or timeout: 30.05 seconds
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(Unknown Source)
	at java.lang.reflect.Constructor.newInstance(Unknown Source)
	at org.openqa.selenium.remote.ErrorHandler.createThrowable(ErrorHandler.java:215)
	at org.openqa.selenium.remote.ErrorHandler.throwIfResponseFailed(ErrorHandler.java:167)
	at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:671)
	at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:410)
	at org.openqa.selenium.remote.RemoteWebDriver.findElementById(RemoteWebDriver.java:453)
	at org.openqa.selenium.By$ById.findElement(By.java:218)
	at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:402)
	at package1.Dependency.login(Dependency.java:27)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
Caused by: org.openqa.selenium.NoSuchElementException: Unable to locate element: {"method":"id","selector":"Email"}
For documentation on this error, please visit: http://seleniumhq.org/exceptions/no_such_element.html
Build info: version: '3.4.0', revision: 'unknown', time: 'unknown'
System info: host: 'SILCP0102', ip: '192.168.100.87', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '1.8.0_73'
Driver info: driver.version: unknown
	at <anonymous class>.FirefoxDriver.prototype.findElementInternal_(file:///C:/Users/SOUMYA~1.BIL/AppData/Local/Temp/anonymous5508931468252070665webdriver-profile/extensions/fxdriver@googlecode.com/components/driver-component.js:11390)
	at <anonymous class>.fxdriver.Timer.prototype.setTimeout/<.notify(file:///C:/Users/SOUMYA~1.BIL/AppData/Local/Temp/anonymous5508931468252070665webdriver-profile/extensions/fxdriver@googlecode.com/components/driver-component.js:792)
]]>
            </full-stacktrace>
          </exception> <!-- org.openqa.selenium.NoSuchElementException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- login -->
        <test-method status="SKIP" signature="validation()[pri:2, instance:package1.Dependency@5e5792a0]" name="validation" duration-ms="0" started-at="2017-10-17T15:07:49Z" depends-on-methods="package1.Dependency.login" finished-at="2017-10-17T15:07:49Z">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method Dependency.validation()[pri:2, instance:package1.Dependency@5e5792a0] depends on not successfully finished methods]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method Dependency.validation()[pri:2, instance:package1.Dependency@5e5792a0] depends on not successfully finished methods
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1084)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- validation -->
        <test-method status="PASS" signature="ending()[pri:3, instance:package1.Dependency@5e5792a0]" name="ending" duration-ms="3" started-at="2017-10-17T15:07:49Z" finished-at="2017-10-17T15:07:49Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- ending -->
      </class> <!-- package1.Dependency -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
